expyfun/_eyelink_controller.py

```diff
@@ -615,9 +615,17 @@ def custom_calibration(self, ctype='HV5', horiz=2./3., vert=2./3.,
         if ctype not in allowed_types:
             raise ValueError('ctype cannot be "{0}", but must be one of {1}'
                              ''.format(ctype, allowed_types))
-        if ctype != 'custom' and not coordinates == None:
-            raise ValueError('If ctype is \'custom\' coordinates must be '
-                             'specified.')
+        if ctype != 'custom':
+            if not coordinates == None:
+                raise ValueError('If ctype is not \'custom\' coordinates canno'
+                                 't be used to generate calibration pattern.')
+        else:
+            if isinstance(coordinates, list):
+                raise ValueError('Coordinates must be in a list.')
+            if any([not isinstance(c, list) or len(c) != 2 
+                    for c in coordinates]):
+                raise ValueError('Each coordinate must be a list with length 2'
+                                '.')
```

Comment:
Should there also be a stipulation that there should be a minimum number of coordinates? I'm assuming that the eyetracker can't do much with less than 3.